docs/source/tutorial/betweenness_centrality.rst

```diff
@@ -0,0 +1,117 @@
+===================================
+Working with Betweenness Centrality
+===================================
+
+Betweenness centrality is the measure of centrality in a graph based on shortest
+paths. For every pair of nodes in a connected graph there is at least a single
+shortest path between the nodes such that the number of edges the path passes
+through is minimized. The betweenness centrality for each node is the number
+of these shortest paths that pass through the node.
+
+This is defined as:
+
+.. math::
+
+    c_B(v) =\sum_{s,t \in V} \frac{\sigma(s, t|v)}{\sigma(s, t)}
+
+where :math:`V` is the set of nodes, :math:`\sigma(s, t)` is the number of
+shortest :math:`(s, t)` paths, and :math:`\sigma(s, t|v)` is the number of
+those paths passing through some  node :math:`v` other than :math:`s, t`.
+If :math:`s = t`, :math:`\sigma(s, t) = 1`, and if :math:`v \in {s, t}`,
+i:math:`\sigma(s, t|v) = 0`
+
+This tutorial will take you through the process of calculating the betweenness
+centrality of a graph and visualizing it.
+
+Generate a Graph
+----------------
+
+To start we need to generate a graph to
+
+.. jupyter-execute::
+
+    import retworkx as rx
+    from retworkx.visualization import mpl_draw
+
+    graph = rx.generators.hexagonal_lattice_graph(4, 4)
+    mpl_draw(graph)
+
+
+Calculate the Betweeness Centrality
+-----------------------------------
+
+The :func:`~retworkx.betweenness_centrality` function can be used to calculate
+the betweenness centrality for each node in the graph.
+
+.. jupyter-execute::
+
+    import pprint
+
+    centrality = rx.betweenness_centrality(graph)
+    # Print the centrality value for the first 5 nodes in the graph
+    pprint.pprint({x: centrality[x] for x in range(5)})
+
+The output of the :func:`~retworkx.betweenness_centrality` is a
+:class:`.CentralityMapping` which is a custom
+`mapping <https://docs.python.org/3/glossary.html#term-mapping>`__ type that
+maps the node index to the centrality value as a float. This is a mapping and
+not a sequence because node indices (and edge indices too, which is not
+relevant here) are not guaranteed to be a contiguous sequence if there are
+removals.
+
+Visualize the Betweenness Centrality
+------------------------------------
+
+Now that we've found the betweenness centrality for ``graph`` lets visualize it.
+We'll color each node in the output visualization using its calculated
+centrality:
+
+.. jupyter-execute::
+
+    import matplotlib.pyplot as plt
+
+    # Generate a color list
+    colors = []
+    for node in graph.node_indices():
```

Comment:
Just curious, was `node_indexes` replaced by `node_indices` in the development version or is this a typo?

Reply:
Not replaced, but for this pr (which I'll probably split it out prior to removing the wip tag) I added an alias method so you can call either `node_indexes` or `node_indices` in preparation for a future deprecation and eventual removal as part of a 1.0.0 release.

Reply:
I split it out in: https://github.com/Qiskit/retworkx/pull/541